src/components/cards/NavMenuCard/NavMenuCard.tsx

```diff
@@ -175,32 +187,58 @@ function NavMenuCard() {
                           }}
                         />
                       );
-                  })
-                : Object.entries(ROUTES).map(([key, route]) => {
-                    if (route.protected && walletAddress)
-                      return (
-                        <NavBarLink
-                          path={route.path}
-                          linkText={route.text}
-                          key={key}
-                          onClick={() => {
-                            setShowMenu(false);
-                          }}
-                        >
-                          <>
-                            {route.icon ? (
-                              route.icon({
-                                height: 24,
-                                width: 24,
-                                fill: 'var(--text-white)',
-                              })
-                            ) : (
-                              <></>
-                            )}
-                          </>
-                        </NavBarLink>
-                      );
                   })}
+                  <NavBarLink
+                    path={''}
+                    linkText={'Create'}
+                    key={'create-ant-nav-button'}
+                    onClick={
+                      walletAddress
+                        ? () =>
+                            dispatchGlobalState({
+                              type: 'setShowCreateAnt',
+                              payload: true,
+                            })
+                        : () => {
+                            dispatchGlobalState({
+                              type: 'setShowCreateAnt',
+                              payload: true,
+                            });
+                            dispatchGlobalState({
+                              type: 'setShowConnectWallet',
+                              payload: true,
+                            });
```

Comment:
nit: remove these double dispatches now that CreateAntModal manages showing connect wallet modal

Reply:
woops, removed